### ISSUE TITLE
Remove QgsWkbTypes::Type::xxxx25D

### ISSUE THREAD

**Issue by rouault** [qgis/qgis3.0_api#107]
I don't think we need those xxxx25D enumeration values that are duplicate of the xxxxZ variant. At least not in the public API (if we need them in a few spots, then just move them to a dedicated enumeration). This is a constant source of confusion & bugs (like https://github.com/qgis/QGIS/issues/33883)

**Comment by nyalldawson** [comment]
Does GDAL still use these?

**Comment by rouault** [comment]
> Does GDAL still use these?

yes. But that could perhaps be isolated in QGIS to OGR-only code paths ? What is confusing in QGIS is the presence of both xxx25D and xxxZ.

**Comment by nyalldawson** [comment]
Fair enough!
